src/webargs/flaskparser.py

```diff
@@ -48,17 +49,20 @@ class FlaskParser(core.Parser):
     """Flask request argument parser."""
 
     __location_map__ = dict(
-        view_args="parse_view_args",
-        path="parse_view_args",
+        view_args="location_load_view_args",
+        path="location_load_view_args",
         **core.Parser.__location_map__
     )
 
-    def parse_view_args(self, req, name, field):
-        """Pull a value from the request's ``view_args``."""
-        return core.get_value(req.view_args, name, field)
+    def location_load_view_args(self, req, schema):
+        """Pull a the request's ``view_args``."""
+        return req.view_args or core.missing
 
-    def parse_json(self, req, name, field):
+    def location_load_json(self, req, schema):
```

Comment:
How about `get_json` etc.? It's less verbose and no less clear, IMO.

Reply:
Yeah, I'm fine with that. But I think I want to keep the name `_get_loader` for the wrapper which looks up these methods in the `__location_map__`. I can't think of a better name for that.

Another option is `load_json` if you like that. I'll rename to `get_{location}` throughout and I don't mind doing another rename if you want.

Reply:
Actually, as I was in the midst of doing this, I saw that there are other functions like `get_default_request` and `get_request_from_view_args` which also have `get_` as a prefix. I'm going to go with `load_*` for now and we can discuss changing it. I'd like for the data-reading functions to have a distinct shared prefix that makes them identifiable.

Reply:
Fine with me